### PR TITLE
liquidity: update defaults budget and confirmation target

### PR DESCRIPTION
**Pull request by carlaKC**
This PR increases the confirmation target for automated loop out swaps and bumps our liquidity budget to use our new percentage based fee scheme.

#### Pull Request Checklist
- [x] Update `release_notes.md` if your PR contains major features, breaking changes or bugfixes

### CODE REVIEW

File: liquidity/liquidity.go [hunk 1]
```diff
@@ -83,12 +83,9 @@ var (
...
 	// amount chosen simply uses the current defaults to provide budget for
-	// a single swap. We don't have a swap amount to calculate our maximum
-	// routing fee, so we use 0.16 BTC for now.
-	defaultBudget = defaultMaximumMinerFee +
-		ppmToSat(funding.MaxBtcFundingAmount, defaultSwapFeePPM) +
-		ppmToSat(defaultMaximumPrepay, defaultPrepayRoutingFeePPM) +
-		ppmToSat(funding.MaxBtcFundingAmount, defaultRoutingFeePPM)
+	// a single swap. We don't have a swap amount so we just use our max
+	// funding amount.
+	defaultBudget = ppmToSat(funding.MaxBtcFundingAmount, defaultFeePPM)
```
Comment (guggero) [inline liquidity/liquidity.go:93]: Just out of curiosity: did the default budget increase or decrease with this change?

**Review by guggero — APPROVED** [review]
LGTM :heavy_check_mark:

**Review by bhandras — APPROVED** [review]
LGTM 🌊

File: liquidity/liquidity.go (hunk 1 above)
Comment (carlaKC) [reply liquidity/liquidity.go:93]: Gone down a bit 💰

**Pull request merged**